CHANGES

```diff
@@ -9,15 +9,6 @@
 
  Changes between 1.1.1c and 1.1.1d [xx XXX xxxx]
 
-  *) Early start up entropy quality from the DEVRANDOM seed source has been
-     improved for older Linux systems.  The RAND subsystem will wait for
-     /dev/random to be producing output before seeding from /dev/urandom.
-     The seeded state is stored for future library initialisations using
-     a system global shared memory segment.  The shared memory identifier
-     can be configured by defining OPENSSL_RAND_SEED_DEVRANDOM_SHM_ID to
```

Comment:
Shouldn't we be keeping this text and deleting the other one? This one refers to `OPENSSL_RAND_SEED_DEVRANDOM_SHM_ID` which the other refers to `OSSL_SHM_SEEDED`

Reply:
Yeah, updated.  I need a rest...